docs/usage.md

```diff
@@ -222,24 +222,24 @@ Otherwise, you can provide your own position file in the `--steps impute` with S
 nextflow run nf-core/phaseimpute --input samplesheet.csv --steps impute --posfile samplesheet_posfile.csv  --tool stitch --outdir results --genome GRCh37 -profile docker
 ```
 
-The csv provided in `--posfile` must contain two columns [chr, file]. The first column is the chromosome and the file column are tsvs with the list of positions, unique to each chromosome.
+The csv provided in `--posfile` must contain four columns [panel, chr, vcf, txt]. The first column [panel] is a name to identify the sites, typically a panel name. The second column [chr] is the chromosome corresponding to that file. The third column [vcf] can be kept empty. Lastly, the fourth column [txt] is a compressed tsv file containing the list of positions, unique to each chromosome.
```

Comment:
We should add a bit more informations about the third column.
This vcf is used for the computation of genotype likelihood in the preprocess of glimpse1 and in the validation step.